cmake/Modules/Findfmt.cmake

```diff
@@ -0,0 +1,64 @@
+add_library(fmt INTERFACE IMPORTED)
+
+set(fmt_external_project_name fmtlib_fmt)
+
+find_path(fmt_INCLUDE_DIR fmt/format.h
+  PATHS ${EP_PREFIX}/src/${fmt_external_project_name}/include)
+mark_as_advanced(fmt_INCLUDE_DIR)
+
+# read the fmt version stored in fmt/core.h
+unset(fmt_VERSION)
+set(fmt_version_file_path ${fmt_INCLUDE_DIR}/fmt/core.h)
```

Comment:
That makes me cry :( Maybe move to conan? or at least to hunter

Reply:
In a separate PR, please!